pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/BaseTransformFunction.java

```diff
@@ -324,6 +339,17 @@ public String[] transformToStringValuesSV(ProjectionBlock projectionBlock) {
           BigDecimal[] bigDecimalValues = transformToBigDecimalValuesSV(projectionBlock);
           ArrayCopyUtils.copy(bigDecimalValues, _stringValuesSV, length);
           break;
+        case BOOLEAN:
+          intValues = transformToIntValuesSV(projectionBlock);
+          for (int i = 0; i < length; i++) {
+            _stringValuesSV[i] = Boolean.toString(intValues[i] == 1);
+          }
+          break;
+        case TIMESTAMP:
+          longValues = transformToLongValuesSV(projectionBlock);
+          for (int i = 0; i < length; i++) {
+            _stringValuesSV[i] = new Timestamp(longValues[i]).toString();
+          }
```

Comment:
missing break;

Reply:
We don't really need break for default branch. Changed it to if-else branches for clarity

Reply:
Oh, we do have linter check for that lol